skills/hears.js

```diff
@@ -3,9 +3,12 @@ const moment = require('moment');
 const { Skill, Message, Point } = require('../models/schema');
 const personService = require('../lib/personService');
 const genAsyncBot = require('../lib/asyncBot');
+const { extractMentionedPeople } = require('../lib/message');
 const { LEARNING_KEY, MAIN_HELP_TEXT } = require('../static');
 
-const extractSkills = messageString => {
+const LEARNING_KEY = ':tanabata_tree:';
+
+const extractSkills = (messageString) => {
```

Comment:
@ZhangChen199102 should we move `extractSkills` to a separate file in `/lib` directory to be consistent with `extractMentionedPeople`?

Reply:
@phoebe-lew yeah sounds good, will do.